install.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+if [[ -n $1 ]] && [[ $1 == '--scripts' ]]; then
+  echo '💡  Installing scripts…'
+
+  PATH_LINE='export PATH=$PATH:$HOME/.git-semantic-commits:$PATH'
+  BASHRC=~/.bashrc
+  ZSHRC=~/.zshrc
+
+  if [ -f $BASHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $BASHRC
+    then
+      echo
+      echo "📝  Adding path to $BASHRC"
+      echo $PATH_LINE >> $BASHRC
+      source $BASHRC
+    fi
+  else
+    echo 'NOPE'
```

Comment:
Oh, forgot to delete this debug line! 😞 "Else" statement obviously isn't  needed at all.